### ISSUE TITLE
pandas reset_index() not allowing lists of index levels to remove

### ISSUE THREAD

**Issue by Dr-Irv** [microsoft/pylance-release#411]
-   Language Server version: 2020.9.6
-   OS and version: Windows 10
-   Python version (& distribution if applicable, e.g. Anaconda): Anaconda Python 3.7.5

## Expected behaviour

No error

## Actual behaviour

```python
import pandas as pd
df = pd.DataFrame.from_records([["a", "b", 3], ["a", "c", 4]], columns=["x", "y", "z"])
ndf: pd.DataFrame = df.set_index(["x", "y"])
ndf = ndf.reset_index(["x"])

s: pd.Series = ndf["z"]
sndf = s.reset_index(["x"])
```
reports errors:
```
No overloads for "ndf.reset_index(["x"])" match parameters
  Argument types: (List[str])

No overloads for "s.reset_index(["x"])" match parameters
  Argument types: (List[str])
```

Possible fix:
```
diff pandas/core/frame.pyi.ORIG pandas/core/frame.pyi
1230c1230
<         level: Level = ...,
---
>         level: Union[Level, List[Level]] = ...,
1240c1240
<         level: Level = ...,
---
>         level: Union[Level, List[Level]] = ...,
1250c1250
<         level: Level = ...,
---
>         level: Union[Level, List[Level]] = ...,
1259c1259
<         level: Level = ...,
---
>         level: Union[Level, List[Level]] = ...,
```
and
```
diff pandas/core/series.pyi.ORIG pandas/core/series.pyi
191c191
<         self, level: Optional[Level], drop: Literal[True], name: Optional[object] = ..., inplace: _bool = ...,
---
>         self, level: Optional[Union[Level,List[Level]]], drop: Literal[True], name: Optional[object] = ..., inplace: _bool = ...,
195c195
<         self, level: Optional[Level] = ..., name: Optional[object] = ..., inplace: _bool = ..., *, drop: Literal[True]
---
>         self, level: Optional[Union[Level,List[Level]]] = ..., name: Optional[object] = ..., inplace: _bool = ..., *, drop: Literal[True]
199c199
<         self, level: Optional[Level] = ..., drop: _bool = ..., name: Optional[object] = ..., inplace: _bool = ...,
---
>         self, level: Optional[Union[Level, List[Level]]] = ..., drop: _bool = ..., name: Optional[object] = ..., inplace: _bool = ...,
```

**Comment by gramster** [comment]
Thanks again for these great bug reports!

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.9.7, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#202097-30-september-2020
